doc/pages/administrator/030-installation_kubernetes.md

```diff
@@ -1,10 +1,10 @@
 # Installing Astarte Operator
 
-The most simple and common installation procedure exploits the [Astarte Operator's Helm
```

Comment:
Something went wrong with the diff for this whole file. It is good as it is without applying the patch.